src/app/components/ConfirmOrCancel/index.tsx

```diff
@@ -29,8 +29,10 @@ export default function ConfirmOrCancel({
           onClick={onCancel}
           label={tCommon("actions.cancel")}
           halfWidth
+          disabled={loading}
         />
         <Button
+          type="submit"
```

Comment:
Just a sidenote:
Type "submit" makes only sense when wrapped in a form element, right?
In this PR it is the first time that it is wrapped in a form.
In all other places where this component is used it is not wrapped in a form.
I think it is ok when it remains "submit" for all other occurrences too, but just want to point to that.

Reply:
Yeah, good point. I thought about if it's an issue in other occurrences but I think we're save.  
<img width="331" alt="image" src="https://user-images.githubusercontent.com/1016218/195490501-225bdea3-f0b6-4494-a53b-bddff3d76622.png">
Looking at the current usage we could also create a follow-up issue to switch all screens to use forms/submit as well. wdyt?

Reply:
Yeah that would be great. Actually I feel most users are tempted to press enter after filling form rather than clicking confirm. I feel it will improve user experience and also code readability.

Reply:
@lisabaut in this case I would vote for `optional` `onConfirm`. This will support current usages and provides the flexibility in the future to also use it in a non-form context. wdyt?

Reply:
Fine for me

Reply:
We reached an agreement! 🎉

Reply:
🎉